src/Shared/Application/Command/SchemathesisUserSeeder.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Shared\Application\Command;
+
+use App\Shared\Application\Fixture\SchemathesisFixtures;
+use App\Shared\Infrastructure\Transformer\UuidTransformer;
+use App\User\Domain\Entity\User;
+use App\User\Domain\Entity\UserInterface;
+use App\User\Domain\Factory\UserFactoryInterface;
+use App\User\Domain\Repository\UserRepositoryInterface;
+use Symfony\Component\PasswordHasher\Hasher\PasswordHasherFactoryInterface;
+
+final readonly class SchemathesisUserSeeder
+{
+    private const USER_DEFINITIONS = [
+        'primary' => [
+            'id' => SchemathesisFixtures::USER_ID,
+            'email' => SchemathesisFixtures::USER_EMAIL,
+            'initials' => SchemathesisFixtures::USER_INITIALS,
+            'confirmed' => false,
+        ],
+        'update' => [
+            'id' => SchemathesisFixtures::UPDATE_USER_ID,
+            'email' => SchemathesisFixtures::UPDATE_USER_EMAIL,
+            'initials' => SchemathesisFixtures::UPDATE_USER_INITIALS,
+            'confirmed' => false,
+        ],
+        'delete' => [
+            'id' => SchemathesisFixtures::DELETE_USER_ID,
+            'email' => SchemathesisFixtures::DELETE_USER_EMAIL,
+            'initials' => SchemathesisFixtures::DELETE_USER_INITIALS,
+            'confirmed' => true,
+        ],
+        'password_reset_request' => [
+            'id' => SchemathesisFixtures::PASSWORD_RESET_REQUEST_USER_ID,
+            'email' => SchemathesisFixtures::PASSWORD_RESET_REQUEST_EMAIL,
+            'initials' => SchemathesisFixtures::PASSWORD_RESET_REQUEST_INITIALS,
+            'confirmed' => true,
+        ],
+        'password_reset_confirm' => [
+            'id' => SchemathesisFixtures::PASSWORD_RESET_CONFIRM_USER_ID,
+            'email' => SchemathesisFixtures::PASSWORD_RESET_CONFIRM_EMAIL,
+            'initials' => SchemathesisFixtures::PASSWORD_RESET_CONFIRM_INITIALS,
+            'confirmed' => true,
+        ],
+    ];
+
+    public function __construct(
+        private UserRepositoryInterface $userRepository,
+        private UserFactoryInterface $userFactory,
+        private PasswordHasherFactoryInterface $hasherFactory,
+        private UuidTransformer $uuidTransformer
+    ) {
+    }
+
+    /**
+     * @return array<string,UserInterface>
+     */
+    public function seedUsers(): array
```

Comment:
Please review this class and search the existing classes and methods that can be re used instead of writing the duplicated logic 
Please follow the DRY principle in the project